src/zc/buildout/easy_install.txt

```diff
@@ -46,7 +46,7 @@ index
    The URL of an index server, or almost any other valid URL. :)
 
    If not specified, the Python Package Index,
-   https://pypi.python.org/simple/, is used.  You can specify an
+   https://pypi.org/simple/, is used.  You can specify an
```

Comment:
I see you're changing the default index URL in the documentation, but I don't see you changing it in the code itself.

(Is the default index URL hardcoded in zc.buildout itself, or does it come from setuptools?)

Reply:
@mgedmin He changed the `default_index_url` in `easy_install.py`.

Reply:
It's here: https://github.com/buildout/buildout/pull/445/files#diff-784f9bb654bbcee96dcbc547bbddd493

Reply:
Whoops!